src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -193,13 +194,21 @@ def safe_for_comment(
     return safe_to_comment, filtered_pr_files
 
 
-def get_pr_filenames(pr_files: JSONData) -> List[str]:
+def get_pr_filenames_and_patches(pr_files: JSONData) -> Tuple[List[str], List[str]]:
```

Comment:
They fixed the original issue of why we made `JSONData = Any`, so should type this 
```suggestion
def get_pr_filenames_and_patches(pr_files: Sequence[Mapping[str, str]]) -> Tuple[List[str], List[str]]:
```